tests/test_datetime.py

```diff
@@ -31,8 +31,8 @@
 from tests.conftest import env_var_2_bool
 
 ABSOLUTE_TIME_SPECS = [
-    dtm.datetime.now(tz=dtm.timezone(dtm.timedelta(hours=-7))),
-    dtm.datetime.utcnow(),
+    dtm.datetime.now(tz=dtm.timezone(dtm.timedelta(hours=-7))).replace(second=0, microsecond=0),
+    dtm.datetime.utcnow().replace(second=0, microsecond=0),
```

Comment:
why?

Reply:
when initializing pytest, each worker got a slightly different value for this, hence it showed an error saying that it got different items in this list.

Reply:
ah, interesting!